File: try/uppercase.c
#include<stdio.h>
#include<cs50.c>
#include<string.h>
#include<ctype.h>

int main(void)
{
    string s = get_string("", " before: ");
    printf("AFTER: ");

    for (int i = 0, n = strlen(s); i < n; i++)
    {
            printf("%c", toupper(s[i]));
    }
    printf("\n");
}

File: try/uppercase.py
from cs50 import get_string

before = get_string("Before: ")
print(f"After: {before.upper()}")

#----OR----

print("After: ", end="")
for c in before:
    print(c.upper(), end="")
print()

#----OR----

print("After: " + before.upper())